.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,42 @@
+name: auth-landing CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm ci
+      - name: Format code
+        run: npm run check-format
+      - name: Lint svelte files
+        run: npm run lint
+      - name: Generate production build
+        run: npm run export
+  deploy-master:
+    if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+    needs: test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+    steps:
```

Comment:
We'll need the `checkout`, `setup-node`, and export actions here, since the filesystem are not shared across jobs.

Also, I suppose `matrix` is unneeded, since we can just inline the version number into the parameters of `setup-node` action.

Reply:
gotcha, I'll duplicate those actions